WifiART/src/structs/AltitudeType.java

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package structs;
+
+/**
+ * An enum representing the altitude type options for the LCI subelement.
+ */
+public enum AltitudeType {
+    NO_KNOWN_ALTITUDE(0),
+    ALTITUDE_IN_METERS(1),
+    ALTITUDE_IN_FLOORS(2);
+
+    private final int value;
+
+    AltitudeType(int value) {
```

Comment:
nit: should the constructor be private? The constructor should not be used outside of the AltitudeType class.

Reply:
Good catch.

Reply:
On second thought, I think enum constructors are private by necessity, so the "private" access modifier would be redundant and including it is just a matter of preference.

Reply:
Ah good point. Fine with leaving it out then.